tests/libjlm/backend/hls/rvsdg2rhls/TestTheta.cpp

```diff
@@ -9,6 +9,7 @@
 #include <jive/rvsdg/theta.hpp>
 
 #include <jlm/backend/hls/rvsdg2rhls/rvsdg2rhls.hpp>
```

Comment:
The same here. Could this header be removed?

Reply:
Removed